helm/templates/_cm_helpers.tpl

```diff
@@ -0,0 +1,80 @@
+{{/*
```

Comment:
we've talked a bit about breaking the _helpers.tpl down into smaller chunks. this is an effort to start that